src/imageio.cs

```diff
@@ -724,6 +740,270 @@ interface CGImageProperties {
 		[Field ("kCGImagePropertyDNGFixVignetteRadial")]
 		NSString DNGFixVignetteRadial { get; }
 
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGActiveArea")]
+		NSString DNGActiveArea { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGAnalogBalance")]
+		NSString DNGAnalogBalance { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGAntiAliasStrength")]
+		NSString DNGAntiAliasStrength { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGAsShotICCProfile")]
+		NSString DNGAsShotICCProfile { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGAsShotPreProfileMatrix")]
+		NSString DNGAsShotPreProfileMatrix { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGAsShotProfileName")]
+		NSString DNGAsShotProfileName { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGBaselineExposureOffset")]
+		NSString DNGBaselineExposureOffset { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGBayerGreenSplit")]
+		NSString DNGBayerGreenSplit { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGBestQualityScale")]
+		NSString DNGBestQualityScale { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGBlackLevelDeltaH")]
+		NSString DNGBlackLevelDeltaHorizontal { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGBlackLevelDeltaV")]
+		NSString DNGBlackLevelDeltaVertical { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGBlackLevelRepeatDim")]
+		NSString DNGBlackLevelRepeatDim { get; }
+
+		[Watch (5,0), TV (12,0), Mac (10,14, onlyOn64: true), iOS (12,0)]
+		[Field ("kCGImagePropertyDNGCFALayout")]
+		NSString DNGCFALayout { get; }
```

Comment:
For consistency with old API the `DNG` prefix is fine - but the `CFA` one should be `Cfa` (and run tests for typos)